.github/workflows/sync.yml

```
@@ -0,0 +1,33 @@
+name: Synchronize upstream
```

```suggestion
name: Synchronize Upstream
```

Actually, let's stick to our naming convention (All capitalized letters) :)